tests/integration/test_fenicsxprecice.py

```diff
@@ -136,11 +136,11 @@ def right_boundary(x): return abs(x[0] - 1.0) < 10**-14
 
         precice = fenicsxprecice.Adapter(MPI.COMM_WORLD, self.dummy_config)
         precice._participant = Participant(None, None, None, None)
-        precice.initialize(right_boundary, self.scalar_V, self.scalar_function)
+        precice.initialize({"Dummy-Mesh": [right_boundary, self.scalar_V, self.scalar_function]})
```

Comment:
The dictionary is an appropriate data structure. We could also (not necessarily in this PR, because this will tunr it into a never-ending-story) introduce a class `CouplingMesh`. This would allow defining a constructor `CouplingMesh(name="Dummy-Mesh", domain=right_boundary, function_space=self.scalar_V, fenics_function=self.scalar_function)`. `precice.initialize` would then accept either a single `CouplingMesh` or a collection of `CouplingMesh`.